content/en/blog/2023/kubecon-eu.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Join us for OpenTelemetry Talks and Activities at Kubecon EU 2023
+linkTitle: Kubecon EU '23
+date: 2023-03-30
+spelling:
+  cSpell:ignore Severin Neumann Pavol Loffay Kowall Aiven Vider Xiaochun Alolita
+  Sharma Dyla Benedikt Bongartz Oliveira Pathak
+author: '[Severin Neumann](https://github.com/svrnm)'
+---
+
+The OpenTelemetry project maintainers, and members of the governance committee
+and technical committee are excited to be at [Kubecon EU][] in Amsterdam from
+April 18 - 21, 2023!
+
+You can meet us at the [project booth][], listen to our talks or learn with us
+during workshops. We are looking forward to hear from you, how you are using
+OpenTelemetry or contributing to the project.
+
+Below you will find a list of talks and other activities related to
+OpenTelemetry.
+
+## Kubecon Talks and Maintainer Sessions
+
+- **[Jaeger: The Future with OpenTelemetry and Metrics](https://sched.co/1HySf)**<br>
+  by Pavol Loffay, Red Hat & Jonah Kowall, Aiven<br> Wednesday, April 19 •
+  11:55 - 12:30
+- **[Using OpenTelemetry for Application Security, with a Real Life Example](https://sched.co/1HyYT)**<br>
+  by Ron Vider, Oxeye<br> Wednesday, April 19 • 11:55 - 12:30
+- **[Understand Systems with OpenTelemetry: A Hybrid Telemetry Data Backend](https://sched.co/1Hyb2)**<br>
+  by Ran Xu, Huawei & Xiaochun Yang, Northeastern University<br> Wednesday,
+  April 19 • 14:30 - 15:05
+- **[OTel Me About Metrics: A Metrics 101 Crash Course](https://sched.co/1Hya1)**<br>
+  by Reese Lee, New Relic<br> Wednesday, April 19 • 17:25 - 18:00
+- **[OpenTelemetry: Using Unified Semantics to Drive Insights + Project Update](https://sched.co/1HyS5)**<br>
+  by Morgan McLean, Splunk, Alolita Sharma, Apple, Daniel Dyla, Dynatrace, & Ted
+  Young, Lightstep<br> Thursday, April 20 • 16:30 - 17:05
+- **[Tutorial: Exploring the Power of OpenTelemetry on Kubernetes](https://sched.co/1HyZ3)**<br>
+  by Pavol Loffay, Benedikt Bongartz & Yuri Oliveira Sa, Red Hat, Severin
+  Neumann, Cisco & Kristina Pathak, LightStep<br> Friday, April 21 • 16:00 -
+  17:30
+
+## Co-located Events
+
+The [OpenTelemetry project meeting](https://sched.co/1JWS7) is to meet and
+network with the OpenTelemetry maintainers and core contributors. This meeting
+will be held 13:00 - 15:00 on Tuesday April 18, 2023.
+
+[Observability Day][] _fosters collaboration, discussion, and knowledge sharing
+of cloud-native observability projects_. This event will be held on April 18,
+2023 from 9:00 - 17:00. There will be several sessions on OpenTelemetry as well.
+
+## OpenTelemetry Project Booth
+
+Drop by and say Hi! at the OpenTelemetry project booth in the Kubecon EU Project
+Pavilion. If you’re lucky, you may even pick up some OpenTelemetry swag!
```

Comment:
oh nice :D

Reply:
@svrnm Thanks for putting this post together. Can you add an image or two from Kubecon NA as well as for OTel swag :-)

Reply:
@alolita happy to do so, but I don't have any images, so if you have some, please send them over and I can add them in :-)

Reply:
I'll leave it like it is for now, if anyone has something to add, we can do this through a follow up PR